server/models/index.ts

```diff
@@ -0,0 +1,25 @@
+import { User } from './users';
+import { Job } from './jobs';
+import { Worker } from './workers';
+import { UserWorker } from './user_worker';
+
+// ? Relations
+
+// * User => UserWorker.
+User.hasMany(UserWorker, { foreignKey: 'user_id', sourceKey: 'id' });
+UserWorker.belongsTo(User, { foreignKey: 'user_id' });
+
+// * Worker => UserWorker.
+Worker.hasMany(UserWorker, { foreignKey: 'worker_id', sourceKey: 'id' });
+UserWorker.belongsTo(Worker, { foreignKey: 'worker_id' });
```

Comment:
do you have to use both?
seems redundant

Reply:
As it seems [here](https://sequelize.org/docs/v6/core-concepts/assocs/#goal) in the documentation, I think I do.